.mergify.yml

```diff
@@ -1,61 +1,73 @@
 queue_rules:
   - name: default
-    conditions:
-      - status-success=build
+    merge_method: squash 
+    queue_conditions:
+      - check-success=build
```

Comment:
One bizarre thing:  I couldn't find the previous "status-success" field anyplace in mergify docs...

Reply:
https://github.com/Mergifyio/mergify/pull/2039 only place I found which had me wondering if something was "off" for a long time :(